src/trusted/service_runtime/nacl_syscall_handlers_gen.py

```diff
@@ -246,7 +246,7 @@
     ('NACL_sys_lind_syscall', 'NaClSysLindSyscall', ['uint32_t callNum', 'uint32_t inNum', 'void *inArgs', 'uint32_t outNum', 'void *outArgs']),
     ('NACL_sys_fork', 'NaClSysFork', []),
     ('NACL_sys_execv', 'NaClSysExecv', ['void *path', 'void *argv']),
-    ('NACL_sys_execve', 'NaClSysExecve', ['void *path', 'void *argv', 'void *envp']),
+    ('NACL_sys_execve', 'NaClSysExecve', ['const char *path', 'void *argv', 'void *envp']),
```

Comment:
You changed the type of path in execve but not execv, and you changed the type of argv in the signature in nacl_syscall_common, but not in here.

Reply:
This ones a little confusing because it doesn't let me do `const char*` or `char* const*` in the framework for the handler gen, so they have to be void. I'll be consistent with voids here in this file but use the actual signatures in syscall common.

Reply:
Then, sorry to be such a pest, that should be commented.